Automattic/wp-calypso
# A4A: initail onboarding Modal

Related to https://github.com/Automattic/jetpack-genesis/issues/355

## Proposed Changes

In order to easier approach this task and not to grow it too big, this presents only basic modal, as well as some refactoring to improve the scalability

## Why are these changes being made?




## Testing Instructions



- Navigate to any `/marketplace` page and check `i` button near Referral toggle.
- Clicking it should open a modal.

## Pre-merge Checklist
